webserver/pkgpkr/webservice/tests/test_selenium.py

```diff
@@ -100,9 +100,9 @@ def test_work_process(self):
         # Check if the user at my repositories page
         self.assertEqual(f'http://localhost:{LoginTest.port}/repositories', self.driver.current_url)
 
-        # The first element from the repos list
+        # The last element from the repos list
         first_repo_ele = self.driver.find_element_by_xpath(
-            "//tbody/tr[2]/td[1]/a")
+            "//tbody/tr[4]/td[1]/a")
```

Comment:
Something like this?

```suggestion
            "//tbody//a[text()='pkgpkr1/express']")
```